docs/documentation/stories/continuous-integration.md

```diff
@@ -23,22 +23,24 @@ set up Circle CI and Travis CI.
 Even though `ng test` and `ng e2e` already run on your environment, they need to be adjusted to
 run in CI environments.
 
-When using Chrome in CI environments it has to be started without sandboxing.
-We can achieve that by editing our test configs.
+We'll use [Headless Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome#cli) in CI environments. In some environments we need to start the browser without
+sandboxing or disable the gpu. Here we'll do both. 
 
-In `karma.conf.js`, add a custom launcher called `ChromeNoSandbox` below `browsers`:
+In `karma.conf.js`, add a custom launcher called `myChromeHeadless` below `browsers`:
 
 ```
 browsers: ['Chrome'],
 customLaunchers: {
-  ChromeNoSandbox: {
-    base: 'Chrome',
-    flags: ['--no-sandbox']
+  myChromeHeadless: {
```

Comment:
Nit: what do you think of `ChromeHeadlessCI` instead? I feel the `my` prefix doesn't say much about what's going on.